### PR TITLE
chore: add eslint perfectionist

### PR DESCRIPTION
**Pull request by marcelkooi**

### CODE REVIEW

File: .eslintrc.cjs [hunk 1]
```diff
@@ -29,20 +30,47 @@ module.exports = {
...
+          "components/**",
+          "e2e/**",
+          "helpers/**",
+          "hooks/**",
+          "pages/**",
+          "styles/**",
         ],
         "newlines-between": "never",
+        order: "asc",
+        "read-tsconfig": true,
```
Comment (mariusc23) [inline .eslintrc.cjs:64]: I saw @camflan removed this on another project and I can't find docs for it. Might not be a valid option? If true can you remove from runbook/template too?

File: .eslintrc.cjs [hunk 2]
```diff
@@ -29,20 +30,47 @@ module.exports = {
...
+          "~/**",
+          "@/**",
+          "__generated__/**",
+          "constants/**",
+          "components/**",
+          "e2e/**",
+          "helpers/**",
+          "hooks/**",
+          "pages/**",
+          "styles/**",
```
Comment (camflan) [inline .eslintrc.cjs:60]: can remove these, not relevant. The `internal-pattern` should contain any aliases/paths/etc that we use. Check tsconfigs, etc for this

File: .eslintrc.cjs (hunk 1 above)
Comment (camflan) [reply .eslintrc.cjs:64]: Yeah, newer versions don't support this, so upgrade and remove :)

**Review by mariusc23 — APPROVED** [review]

File: .eslintrc.cjs [hunk 3]
```diff
@@ -29,20 +30,15 @@ module.exports = {
     "react/jsx-props-no-spreading": 0,
     "react/jsx-wrap-multilines": 0,
     "react/prefer-stateless-function": 0,
-    "import/order": [
+    "import/order": 0,
+    "sort-imports": 0,
```
Comment (camflan) [inline .eslintrc.cjs:34]: I think you can get rid of the rules and the plugins entirely?

**Review by camflan — COMMENTED** [review]
if there are some commits that are entirely perfectionist re-ordering stuff - you can make a new PR that excludes them from git-blame

Or maybe we just squashed and you'd have to exclude it all? Might still be worth it to make history/blame a bit nicer

**Pull request merged**